_includes/icon.liquid

```diff
@@ -0,0 +1,27 @@
+{% assign addIcon = include.icon %}
+<img
+  {% if addIcon contains 'http' %}
+    src="{{ addIcon }}"
+  {% else %}
+    src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/{{ addIcon }}"
```

Comment:
I have a suggestion: what about creating another input to `icon.liquid`, for example `devicon`, and then set a precedence order. For example, use `devicon` if defined, else use `icon` as the src. This way we could even add other possibilities. Maybe also add [simple-icon](https://simpleicons.org/) as another parameter, which is the icon set used by [shields.io](https://shields.io/badges/static-badge).